device_guids.py

```diff
@@ -35,11 +35,13 @@ def get_guid_i(device) :
 	latest_os_version = None
 	
 	for line in p.stdout :
-		sys.stdout.write(line)
-		if (dev_name_regex.match(line) == None) :
+		sys.stdout.buffer.write(line)
```

Comment:
you need  line.decode('utf-8')  here.  since strLine comes after this line. instead of creating strLine, also better to just do line.decode('utf-8') on other instances. that should fix the script and let it pass CI/CD check.

Reply:
We don't need to convert bytes to string here since we can write bytes directly to stdout.